data/dataAccess.js

```diff
@@ -82,13 +82,13 @@ function buildForwardIndex(term){
     var result = {};
     var termCount = 0;
     var hit = false;
-    var searchExpression = new RegExp(term.toString(), "g");  
+    var searchExpression = new RegExp(term.toLowerCase().toString(), "g");  
 
     Object.entries(methods.availableItems).forEach(([key, value]) =>
     {
         hit = searchExpression.test(value.name.toLowerCase()) ||
-            searchExpression.test(value.shortDescription.toLowerCase().match) ||
```

Comment:
this has been left unchanged from previous implementation to use regex match instead of test. has been removed now.